Add voyager verification to the `verify` command (#2566)
foundry-rs/starknet-foundry: docs/src/starknet/verify.md

Reviewer: I think you could maybe extend this so it explains `Voyager` as an another verification provider. 

Maybe also a comment on using `--verifier` flag to select which one would be used.

Author: Let me do that!

Walnut is a tool for step-by-step debugging of Starknet transactions. You can learn more about Walnut here [walnut.dev](https://walnut.dev). Note that Walnut requires you to specify the Starknet version in your `Scarb.toml` config file.

### Voyager

> [Voyager](https://voyager.online) is a block explorer for Starknet. It gives developers immediate insight into the state of the starknetnetwork. It also intends to become the one-stop location for all Starknet statistics. Note that Voyager requires you to specify the Starknet version in your `Scarb.toml` config file.

Reviewer: ```suggestion
[Voyager](https://voyager.online) is a block explorer for Starknet. It gives developers immediate insight into the state of the Starknet network. Note that Voyager requires you to specify the Starknet version in your `Scarb.toml` config file.
```

Reviewer: Let's keep it simple

```shell
$ sncast \
    verify \

>     --contract-address 0x8448a68b5ea1affc45e3fd4b8b480ea36a51dc34e337a16d2567d32d0c6f8b \

Reviewer: Why was this address changed? It doesn't seem like a class hash of any existing contract
